zipkin4net/Criteo.Profiling.Tracing/ClientTrace.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Criteo.Profiling.Tracing.Annotation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Criteo.Profiling.Tracing
+{
+    public class ClientTrace : IDisposable
+    {
+        public Trace Trace { get; private set; }
+
+        public ClientTrace(string serviceName, string rpc)
+        {
+            Trace = Trace.Current.Child();
```

Comment:
The child is already created in TracingHandler as we need to pass the context in the headers. There is an inconsistency here, you should choose to create it here or in TracingHandler but not both